dev/sbt-plugin/src/main/scala-sbt-1.0/com/lightbend/lagom/sbt/DynamicProjectAdder.scala

```diff
@@ -120,7 +121,7 @@ package com.lightbend.lagom.sbt {
 
         // Now we recreate the structure, this is where the structure data is calculated, which evaluates all the settings
         // and works out all the dependencies.
-        val reindexedStructure = sbt.LagomLoad.reapply(newSession.mergeSettings, structureWithNewProject)
+        val reindexedStructure = sbt.LagomLoad.reapply(newSession.mergeSettings, structureWithNewProject, state.globalLogging.full)
```

Comment:
Not sure if it will result in the same logger, but you can use the build logger (`sLog`):

```scala
val buildLogger = extracted.get(Keys.sLog in ThisBuild)
val reindexedStructure = sbt.LagomLoad.reapply(newSession.mergeSettings, structureWithNewProject, buildLogger)
```

Reply:
I have no idea, to be honest.